packages/core-p2p/src/peer.ts

```diff
@@ -2,12 +2,92 @@ import { app } from "@arkecosystem/core-container";
 import { Logger, P2P } from "@arkecosystem/core-interfaces";
 import axios from "axios";
 import dayjs from "dayjs-ext";
+import Joi from "joi";
 import util from "util";
 import { config as localConfig } from "./config";
 import { PeerPingTimeoutError, PeerStatusResponseError, PeerVerificationFailedError } from "./errors";
 import { PeerVerificationResult, PeerVerifier } from "./peer-verifier";
 
 export class Peer implements P2P.IPeer {
+    private static replySchemas: any = {
```

Comment:
Please move the schemas into a separate file.

Reply:
Moved in abb024254.